src/main/java/anichan/parser/AddToWatchlistParser.java

```diff
@@ -16,15 +16,6 @@
     protected static final String TOO_MUCH_ARGUMENTS = "Add To Watchlist command" + TOO_MUCH_FIELDS;
     protected static final String OUT_OF_BOUND_INDEX_ERROR = "Anime ID is invalid!";
```

Comment:
Since these constants are only used in this class, should this be private instead?

Reply:
Yes you are right, I will make amendments to that! :)